### PR TITLE
Test was not remapping remotely-used fixture file

### PR DESCRIPTION
**Pull request by uuf6429**
Title says it all. :)

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -130,7 +130,7 @@ protected function createDriver()
      *
      * @return string
      */
-    protected function mapRemoteFilePath($file)
+    protected static function mapRemoteFilePath($file)
```
Comment (uuf6429) [inline tests/TestCase.php:133]: I had to make this static since the new usecase is from a static context..

**Comment by aik099** [comment]
Good catch, but I have no idea how to test it considering we're using Selenium 2 for testing and planned (but not implemented) to create a separate driver and Selenium 3.

File: tests/TestCase.php (hunk 1 above)
Comment (stof) [reply tests/TestCase.php:133]: this is a BC break

Comment (stof) [reply tests/TestCase.php:133]: Which new use case ?

Comment (stof) [reply tests/TestCase.php:133]: OK, saw it. To me, we should avoid doing such remapping in the data provider (which might involve splitting that case to a dedicated test)

Comment (uuf6429) [reply tests/TestCase.php:133]: In the ideal world, I would agree, but this is a small low-impact change to fix a problem.
Splitting the test can be easily done later (and switching to non-static).

The being said, I just realised this is the first time I'm seeing static data provider - usually I'm making them non-static. Could that also be a solution?

**Comment by uuf6429** [comment]
> Good catch, but I have no idea how to test it considering we're using Selenium 2 for testing and planned (but not implemented) to create a separate driver and Selenium 3.

I'm not sure what you mean - this is used within tests; if they keep on passing all is well. What is a bit confusing, actually, is how MinkSelenium2Driver tests were not failing.

**Comment by aik099** [comment]
> > Good catch, but I have no idea how to test it considering we're using Selenium 2 for testing and planned (but not implemented) to create a separate driver and Selenium 3.
> 
> I'm not sure what you mean - this is used within tests; if they keep on passing all is well. What is a bit confusing, actually, is how MinkSelenium2Driver tests were not failing.

GitHub Tests use Selenium 2. Maybe it's working there.

We don't use Selenium 3 in GitHub tests. Maybe it's failing for Selenium 3 only.

File: tests/TestCase.php (hunk 1 above)
Comment (stof) [reply tests/TestCase.php:133]: PHPUnit has deprecated the usage of non-static providers in PHPUnit 9.6 and forbids them in PHPUnit 10.

Splitting the test is much easier than asking PHPUnit to rollback this change.

Comment (uuf6429) [reply tests/TestCase.php]: I tried splitting the test, but I didn't like the amount of duplicate code it causes, so I'm remapping the file paths based on an if condition. Not ideal IMO, but somewhat cleaner than a duplicate test.

**Comment by uuf6429** [comment]
@aik099 the reason it wasn't failing before is because the MinkSelenium2Driver test shares the fixture path in a way that it doesn't need a remapping, which is also why the variables are not set by default: https://github.com/minkphp/MinkSelenium2Driver/blob/master/phpunit.xml.dist#L24

I could change my current tests in github to also make it work like so, but failing to remap the paths properly (as is the case now) will break the test on Windows (something I already noticed and fixed).

So quite frankly, if the tests are currently passing for everyone and with my changes they'll still be passing, then don't worry about it and just merge. ;)

**Comment by aik099** [comment]
@uuf6429 , I like the current PR code.

**Review by stof — APPROVED** [review]

**Pull request merged**